src/components/GroupButton/SbGroupButton.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="sb-group-button" :class="activeClasses" v-bind="$attrs">
+    <VNodes :vnodes="activeSlots" />
+  </div>
+</template>
+
+<script>
+import { sharedProps } from '../Button/lib'
+
+export default {
+  name: 'SbGroupButton',
+  components: {
+    VNodes: {
```

Comment:
Hey Lisi, what exactly is this `VNodes` component/prop?

Reply:
Hey Caro, VNodes are the internal way of vue to generate components. It's a functional way to handle and filter whatever is passed through a slot